command/resource/resource_test.go

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package resource
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/docker/go-connections/nat"
+	"github.com/mitchellh/cli"
+	"github.com/stretchr/testify/require"
+
+	"github.com/hashicorp/consul/command/resource/apply"
+	libcluster "github.com/hashicorp/consul/test/integration/consul-container/libs/cluster"
+	libtopology "github.com/hashicorp/consul/test/integration/consul-container/libs/topology"
+)
+
+const (
+	GRPC_PORT     = "8502"
+	GRPC_TLS_PORT = "8503"
+)
+
+func TestClientForwardToServer(t *testing.T) {
```

Comment:
I tried to leave the integration test to `consul/test/integration/consul-container/test/resource/resource_test.go` but it doesn't allow me to import `github.com/hashicorp/consul/command/resource/apply`
I leave the test file to the current position, but it throws [error](https://github.com/hashicorp/consul/actions/runs/7699506102/job/20981320918?pr=20347) saying `test/integration/consul-container/libs/utils/version.go:67:7: set.ConsulCE undefined (type topology.Images has no field or method ConsulCE)`
Has anyone ever seen similar issue before? You are more than welcome to give suggestions here!